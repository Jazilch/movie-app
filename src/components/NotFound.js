'use es6';

import React from 'react';
import styled from 'styled-components';

const NotFoundWrapper = styled.div`
  display: flex;
  justify-content: space-evenly;
  align-items: center;
`;

const ContentWrapper = styled.div``;

const Title = styled.h2`
  color: #fff;
  font-size: 28px;
  font-weight: 700;
  margin: 0;
`;

const SubTitle = styled.span`
  color: #fff;
  font-weight: 300;
`;

const NotFound = () => {
  return (
    <NotFoundWrapper>
      <img
        src="https://cdn2.hubspot.net/hubfs/2676636/Awesome%20Logs%20Files/Hanks-640x359.jpg"
        alt="not-found"
        width="500"
      />
      <ContentWrapper>
        <Title>Awww... Dont cry.</Title>
        <br />
        <SubTitle>Its only a 404 Error</SubTitle>
      </ContentWrapper>
    </NotFoundWrapper>
  );
};

export default NotFound;
