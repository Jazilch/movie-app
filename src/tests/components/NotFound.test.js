/* eslint-disable no-undef */

'use es6';

import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import NotFound from '../../components/NotFound';

it('renders the not found page correctly', () => {
  const notfound = shallow(<NotFound />);
  expect(toJson(notfound)).toMatchSnapshot();
});
