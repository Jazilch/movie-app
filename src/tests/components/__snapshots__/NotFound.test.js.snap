// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the not found page correctly 1`] = `
<styled.div>
  <img
    alt="not-found"
    src="https://cdn2.hubspot.net/hubfs/2676636/Awesome%20Logs%20Files/Hanks-640x359.jpg"
    width="500"
  />
  <styled.div>
    <styled.h2>
      Awww... Dont cry.
    </styled.h2>
    <br />
    <styled.span>
      Its only a 404 Error
    </styled.span>
  </styled.div>
</styled.div>
`;
